### PR TITLE
add remaining rlbot interface functions + one example

### PR DESCRIPTION
**Pull request by ehsanul**

### CODE REVIEW

File: src/rlbot_generated.rs [hunk 1]
```diff
@@ -3874,7 +3874,8 @@ pub mod rlbot {
                     .get::<QuickChatSelection>(
                         QuickChat::VT_QUICKCHATSELECTION,
                         Some(QuickChatSelection::Information_IGotIt),
-                    ).unwrap()
+                    )
+                    .unwrap()
```
Comment (ehsanul) [inline src/rlbot_generated.rs]: `cargo fmt` keeps doing this for me, even though the format linting doesn't care either way. I just got tired of undoing it every time!

Comment (ehsanul) [reply src/rlbot_generated.rs]: I see your version of `rustfmt` actually formatted the other (ie better) way. I may have to upgrade or something.

File: examples/simple_flatbuffer.rs [hunk 2]
```diff
@@ -49,11 +49,36 @@ fn get_input(packet: &GameTickPacket) -> ControllerStateArgs {
...
         throttle: 1.0,
         steer: normalize_angle(steer).max(-1.0).min(1.0),
         ..Default::default()
-    }
+    };
+
+    build_player_input(player_index, controller_state_args)
+}
+
+fn build_player_input<'a>(
```
Comment (ehsanul) [inline examples/simple_flatbuffer.rs]: This function is basically the new boilerplate imposed by taking a `&[u8]` as input for the low-level functions. It's not that bad, and not nearly as bad as the `set_game_state` stuff, which is going to be less common.

File: src/rlbot.rs [hunk 3]
```diff
@@ -145,6 +139,55 @@ impl RLBot {
...
+    /// Render a group of lines/text
+    pub fn render_group(&self, render_group_buffer: &[u8]) -> Result<(), RLBotError> {
+        let status = (self.interface.render_group)(
+            render_group_buffer.as_ptr() as *mut c_void,
+            render_group_buffer.len() as c_int,
+        );
+        core_result(status)
+    }
+
+    // TODO pub fn send_chat
```
Comment (ehsanul) [inline src/rlbot.rs]: I just looked at the function signature and decided IDGAF.

File: examples/simple_flatbuffer.rs (hunk 2 above)
Comment (ehsanul) [reply examples/simple_flatbuffer.rs]: Sure, for the high-level interface it's fine to use either.

File: README.md [hunk 4]
```diff
@@ -184,15 +193,32 @@ bindgen \
     --whitelist-function GameFunctions::UpdatePlayerInputFlatbuffer \
     --whitelist-function RenderFunctions::RenderGroup \
     -- \
+    -fdeclspec \
```
[the remark this answers is not in the document]
Comment (ehsanul) [reply README.md]: Yeah, probably another consequence of not being on windows when doing this.

**Comment by ehsanul** [comment]
@whatisaphone Should be good to go now!

**Comment by whatisaphone** [comment]
Looks good, thanks!

**Pull request merged**